src/utils/sites/utils.js

```diff
@@ -55,4 +57,506 @@ const createRepo = async (templateName, ghToken, siteName) => {
   return data
 }
 
-module.exports = { getTemplatesFromGitHub, createRepo, validateTemplate }
+const SITE_NAME_SUGGESTION_SUFFIX_LENGTH = 6
```

Comment:
I'm not sure of the logic behind this length. I've set it to 6 because all the sites created through the Netlify App UI had suffixes of this length.

Nonetheless, this length is valid.